src/coreclr/utilcode/stresslog.cpp

```diff
@@ -840,6 +862,15 @@ void StressLog::LogMsg(unsigned level, unsigned facility, int cArgs, const char*
     STATIC_CONTRACT_FORBID_FAULT;
     STATIC_CONTRACT_SUPPORTS_DAC;
 
+#ifdef MEMORY_MAPPED_STRESSLOG
+    InterlockedIncrement(&StressLog::writerCount);
```

Comment:
I would be very concerned about the performance on many core systems here - in the GC, the GC threads log tons of messages, and having multiple interlocked instructions in the code path may really hurt performance.

Reply:
Actually, would a much simpler approach work where we change the hMapView in the StressLog data structure to a LPVOID? The idea would be that the destructor for StressLog would no longer close the file mapping, instead the OS would do it when it shuts down the process, so it would be guaranteed to happen after all threads in the process have stopped.

Reply:
It worked! I am updating the PR to use the alternative fix.